
const name = 'komra'
try {
    name = 'tomato';
} catch (error) {
    console.log('we get a error now'error);
}
console.log(name);